generators/server/templates/src/main/java/package/security/SecurityUtils.java.ejs

```diff
@@ -0,0 +1,59 @@
+package <%=packageName%>.security;
+
+import com.nimbusds.jose.proc.SecurityContext;
+import io.micronaut.http.context.ServerRequestContext;
+import io.micronaut.security.authentication.Authentication;
+import io.micronaut.security.authentication.UserDetails;
+import io.micronaut.security.utils.SecurityService;
+
+import java.util.Collection;
+import java.util.Optional;
+
+/**
+ * Utility class for Spring Security.
```

Comment:
Is `Spring` here intended? Maybe confusing

Reply:
No, you're right; fixed that, thanks for finding!